PSI_Project/Models/SubjectGoal.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace PSI_Project.Models
+{
+    public class SubjectGoal
+    {
+        public string SubjectId { get; set; }
+        public double TargetHours { get; set; }
+        public double ActualHoursStudied { get; set; }
+        
+        public SubjectGoal(string subjectId, double targetHours)
+        {
+            SubjectId = subjectId;
+            TargetHours = targetHours;
+            ActualHoursStudied = 0; // 0 hours studied on initialization
+        }
+        
+        public struct StudyDuration // 1. Struct used, not implemented yet TODO: view goals implementation which uses this struct
```

Comment:
not sure if it is a good idea to add struct in the class file, especially when we might need to use it in other part of application (i.e. pomodorro) but if I'm wrong, correct me